native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/xmldata-native/resource-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "bundles":[{
+      "name":"MessagesBundle",
+      "locales":[""]
+    }]
+}
```

Comment:
Can we format the json like this :
```json
{
    "bundles": [
        {
            "name": "MessagesBundle",
            "locales": [
                ""
            ]
        }
    ]
}
```

Reply:
Formatted in [1f4e8bf](https://github.com/ballerina-platform/module-ballerina-xmldata/pull/460/commits/1f4e8bf529ea75e856301b7e6a113e5f119f083c)